tsconfig.json

```diff
@@ -1,25 +1,25 @@
 {
   "compilerOptions": {
-    "declaration": true,
+    "outDir": "./dist/",
+    "rootDir": "./src",
+    "forceConsistentCasingInFileNames": true,
+    "lib": ["es2018"],
     "target": "es6",
```

Comment:
I held them side-by-side and the primary difference is that when targeting `esnext` it doesn't need a polyfill for `await`. However it should be noted that postbuild `uglifyjs-folder` is ran over the code completely compressing and mangling it so at the end of the day it's 1 line with the very minimal of things already regardless. To give an idea, this is `index.js` postbuild (when targeting `es6` that is):

![image](https://user-images.githubusercontent.com/4019718/53307049-7faefb00-3894-11e9-9a9c-3d2eeb75e86a.png)

Don't get me wrong - I do not necessarily disagree... but the gain is very minimal.

Reply:
Oh and this would be `index.js` when targeting `esnext`:

![image](https://user-images.githubusercontent.com/4019718/53307166-e4b72080-3895-11e9-8b70-f3a0cf165e60.png)

Reply:
Update: now it uses `terser-folder`

Terser is more up2date than uglifyjs as described [in their description]([npmjs.com/package/terser#why-choose-terser](https://www.npmjs.com/package/terser#why-choose-terser))

Also it is future proofing, especially needed if we're going to target esnext.

Also Also it will be default in webpack starting v5 so like.. community says it all right?

Reply:
Doubt it and if they do they should provide the necessary polyfills themselve through something like coreJS IMHO.

Changed to ESNext so resolved in 7b222c1d647116c78641fa1cea8e8b2578956a2d